src/features/classification/security-controls/SecurityControls.scss

```diff
@@ -1,14 +1,22 @@
+@import '../../../styles/variables';
+
 .bdl-SecurityControls {
     margin: 0;
     list-style: none;
 }
 
+.bdl-IconSecurityClassificationSolid {
```

Comment:
I think this will hide the icon everywhere by default. Intended? Can we choose to simply not render the icon except in the summarized view, instead?

Reply:
Yes this is intended. I keep it this way for in this case showing/hiding icon using styles simplifies the logics and props passing/checking in components.